YololEmulator.Tests/Analysis/Inspection/FindBooleans.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Yolol.Analysis.ControlFlowGraph;
+using Yolol.Analysis.ControlFlowGraph.Extensions;
+using Yolol.Analysis.TreeVisitor.Reduction;
+using Yolol.Grammar;
+using Yolol.Grammar.AST;
+using Yolol.Grammar.AST.Expressions;
+using Yolol.Grammar.AST.Statements;
+
+namespace YololEmulator.Tests.Analysis.Inspection
+{
+    [TestClass]
+    public class FindBooleans
+    {
+        private void _test(Program ast, IEnumerable<VariableName> originalBooleanVariableNames)
```

Comment:
private methods should be named `Test`, only private fields are named with leading underscores.

Reply:
I hadn't noticed that pattern, but I'll change that now.